permission_handler/pubspec.yaml

```
@@ -15,13 +15,13 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.1.6
-  permission_handler_platform_interface: ^2.0.2
+  meta: ^1.3.0
+  permission_handler_platform_interface: ^3.0.0-nullsafety
 
 dev_dependencies:
-  effective_dart: ^1.2.1
-  plugin_platform_interface: ^1.0.1
+  effective_dart: ^1.3.0
+  plugin_platform_interface: ^1.0.3
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```

Update this to the NNBD stable version:

```suggestion
  sdk: ">=2.12.0-259.9.beta <3.0.0"
```